### ISSUE TITLE
Extract information on SQL value functions from Postgres source

### ISSUE THREAD

**Issue by herwinw** [pganalyze/pg_query#106]
https://github.com/Quarantainenet/pg_query/blob/d3c4889ce03ca5b923509de83b60d62675000524/lib/pg_query/deparse.rb#L791

This lookup table is duplicated from the postgres source, and guaranteed to get out of sync at some point in the future. This should be generated from the postgres source instead

(Future work from #103)

**Comment by herwinw** [comment]
There are more issues like this in the deparse source. Some example:

Constraint types
```ruby
  case node['contype']
  when CONSTR_TYPE_NULL
    output << 'NULL'
  when CONSTR_TYPE_NOTNULL
    output << 'NOT NULL'
  when CONSTR_TYPE_DEFAULT
    output << 'DEFAULT'
  ...
```

Vacuum options:
```ruby
  output << 'FULL' if node['options'][4] == 1
  output << 'FREEZE' if node['options'][3] == 1
  output << 'VERBOSE' if node['options'][2] == 1
  output << 'ANALYZE' if node['options'][1] == 1
```

**Comment by lfittl** [comment]
@herwinw This should be addressed with the new 2.0 release, which has full enum types (generated via the Protobuf integration) instead of integer values - the lookup tables are now gone from this library, and instead auto-generated :)
